styleguide/STYLEGUIDE.md

```diff
@@ -3,23 +3,18 @@
 * We primarily use Kotlin and follow coding conventions based on the [Android Kotlin Style Guide](https://developer.android.com/kotlin/style-guide)
 * There may be instances of code that pre-dates our use of this style guide, these can be refactored as we encounter them.
 
-###  Android Studio Setup
+###  Code formatting
 
-We've provided a coding style that can be imported into Android Studio.
-The file can be found [here](ddg-settings.zip).
+To adhere to codestyle, please run `./gradlew spotlessApply` to autoformat in order to fix any CI issues.
 
-* From Android Studio click on File -> Import Settings
+If you want to do this automatically upon commit we recommend the following `pre-commit` hook.
 
-![Setting kotlin style screenshot](import_settings_one.png)
```

Comment:
Can you also delete the DDG.xml along with all the screenshots and the ddg-settings.zip file from the styleguide folder?

Reply:
Done